larpandoracontent/LArThreeDReco/LArHitCreation/RANSACMethod.h

```diff
@@ -0,0 +1,275 @@
+/**
```

Comment:
General note on comments. Prefer @return comments to describe what the return value is conceptually, rather than its type.